_posts/2015-02-09-create-your-application.markdown

```diff
@@ -81,12 +80,8 @@ For more information about generating a Rails application see the [Getting Start
      ```sh
      $ rails g blacklight:install --devise
      ```
-
-  1. Run GeoBlacklight generator (with a jetty instance for Solr)
-
-     ```sh
-     $ rails g geoblacklight:install --jettywrapper
```

Comment:
I think we need to keep the `geoblacklight:install` command here somewhere (but drop the jetty?)